AzureCalling/Assets.xcassets/acsLogo.imageset/Contents.json

```diff
@@ -0,0 +1,23 @@
+{
+  "images" : [
+    {
+      "filename" : "beab6721 a8e3 4493 b90b 351ab8c715db.png",
```

Comment:
Can we update the file name to align with the image set?

Reply:
We can but maybe we shouldn't.
This UUID comes from Figma, so probably helps us track down where the design came from. The filename is not referenced in the code.

Reply:
Done